src/genie/libs/parser/iosxe/show_run.py

```diff
@@ -253,10 +253,27 @@ class ShowRunInterfaceSchema(MetaParser):
                 Optional('snmp_trap_mac_notification_change_removed'): bool,
                 Optional('spanning_tree_bpduguard'): str,
                 Optional('spanning_tree_portfast'): bool,
+				Optional('spanning_tree_bpdufilter'): str,
                 Optional('switchport_access_vlan'): str,
+				Optional('switchport_trunk_vlans'): str,
                 Optional('switchport_mode'): str,
                 Optional('switchport_nonegotiate'): str,
 				Optional('vrf'): str,
+				Optional('channel_group'): {
+						'chg': str,
```

Comment:
channel_group seems to be an "int" why it is a "str" in schema. Kindly change it.